src/zampy/reference/variables.py

```diff
@@ -17,6 +17,7 @@ def unit_registration() -> UnitRegistry:
         "kilogram_per_square_meter_second = kilogram/(meter**2*second)"
     )
     unit_registry.define("milimeter_per_second = watt/meter**2")
+    unit_registry.define("kilogram_per_kilogram = kilogram/kilogram")
```

Comment:
I think this resolves to be unit less. Like the fraction definition: 
```py
    unit_registry.define("fraction = [] = frac")
```

Reply:
Corrected!